src/elfcore.c

```diff
@@ -1806,7 +1859,9 @@ int InternalGetCoreDump(void *frame, int num_threads, pid_t *pids,
   prpsinfo.pr_gid   = sys_getegid();
   prpsinfo.pr_pid   = main_pid;
   prpsinfo.pr_ppid  = sys_getppid();
+  #if !defined(__aarch64__)
```

Comment:
Strange, why wouldn't getpgrp() not be available on ARM?  That's a bog-standard POSIX call that's been around forever.

Reply:
> Strange, why wouldn't getpgrp() not be available on ARM? That's a bog-standard POSIX call that's been around forever.

This is a syscall `sys_getpgrp()` from `LSS`. It is deprecated in the latest `asm-generic/unistd.h` but can be enabled via `__ARCH_WANT_SYSCALL_DEPRECATED`
I will enable it.

Reply:
It turned out that `getpgrp()` always returns the PGID, so don't need to bother with the system calls from LSS and use `getpgrp()` on all platforms.